<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="test"
    errors="0" failures="1" hostname="" id="" package="test" skipped="0"
    tests="1" time="5.786" timestamp="Sun, 23 Sep 2018 19:27:28 GMT">
  
    <testcase name="default e2e tests" classname="test" time="5.786" assertions="19">

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    

      <failure message="Testing if element &lt;.rooms .room&gt; has count: 5">    at Object.test [as default e2e tests] (/Users/mehdi/Tests/StationF-FrontEnd/test/e2e/specs/test.js:37:15)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)</failure>

    
    </testcase>
  

  

  
  </testsuite>
</testsuites>
